pkg/config/nerdctl_config_applier.go

```diff
@@ -9,10 +9,12 @@ import (
 	"path"
 	"strings"
 
+	"github.com/lima-vm/lima/pkg/networks"
```

Comment:
nit: import order

Reply:
How can I know prevent these nit issue? I ran golang-lint and it didnt give me anything

Reply:
Hi @TrungBui59, golang-lint will lint code, but not format it. You can use `gofmt -w .` from the top level directory of a Go project to format all Go files, writing the correctly formatted files. Go plugins for modern IDEs will typically do this every time a Go file is saved.

See https://go.dev/blog/gofmt